typescript/cli/src/commands/warp.ts

```diff
@@ -89,27 +86,36 @@ export const warpCommand: CommandModule = {
   handler: () => log('Command required'),
 };
 
-export const apply: CommandModuleWithWarpApplyContext<{
-  config?: string;
-  warp?: string;
-  symbol?: string;
-  warpRouteId?: string;
-  strategy?: string;
-  receiptsDir: string;
-}> = {
+const SELECT_WARP_ROUTE_BUILDER = {
```

Comment:
wonder if this should be in https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/cli/src/commands/options.ts ?

but if there's no other commands that might want to reuse this, happy to leave this here

Reply:
will leave in here for now